src/components/LoanCards/LoanMatchingText.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<span class="tw-flex tw-items-center">
+		<span
+			class="
+				tw-text-h4
+				tw-mr-1
+				tw-pl-0.5
```

Comment:
Rather then padding left to get the emoji in the right spot, would text-center get you there?